adapter/internal/api/apis_impl.go

```diff
@@ -115,13 +120,22 @@ func extractAPIProject(payload []byte) (apiProject ProjectAPI, err error) {
 				loggers.LoggerAPI.Errorf("Error occured converting api file to json: %v", conversionErr.Error())
 				return apiProject, conversionErr
 			}
-			apiType, err := getAPIType(apiJsn)
-			if err != nil {
-				loggers.LoggerAPI.Errorf("Error occured while reading the api type : %v", err.Error())
+
+			apiType, lifeCycleStatus, productionEndpoint, sandboxEndpoint, endpointImplementationType, err :=
+				extractAPIInformation(apiJsn)
+
+			if endpointImplementationType == "INLINE" {
+				errMsg := "Inline endpointImplementationType is not currently supported with WSO2 micro-gateway"
+				loggers.LoggerAPI.Infof(errMsg)
+				err = errors.New(errMsg)
 				return apiProject, err
 			}
 			apiProject.APIJsn = apiJsn
 			apiProject.APIType = apiType
+			apiProject.APILifeCycleStatus = lifeCycleStatus
+			apiProject.ProductionEndpoint = productionEndpoint
+			apiProject.SandboxEndpoint = sandboxEndpoint
+			apiProject.APIJsn = apiJsn
```

Comment:
```suggestion
		
```
Seems this is an additional line

Reply:
fixed by https://github.com/wso2/product-microgateway/pull/1682/commits/459a83228043b61d1de8f7be99d9a054fefb1cbd